### PR TITLE
SEO-176062-Windowsforms-duplicate-titles

### PR DESCRIPTION
**Pull request by Edithabich**
HI @MallikaRK ,
Have updated UG titles please review.
Thanks.

Regards,
Edith Abich.

### CODE REVIEW

File: WindowsForms/Classic/DateTimePicker/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms xptoolbar control | Syncfusion
+title: Overview of Windows Forms DateTimePicker control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/Classic/DateTimePicker/Overview.md:3]: Control

File: WindowsForms/MessageBox/Overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Overview of MessageBoxAdv control | Windows Forms | Syncfusion
+title: Overview of Windows Forms MessageBoxAdv control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/MessageBox/Overview.md:3]: Overview of Windows Forms MessageBox Control | Syncfusion

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/MessageBox/Overview.md (hunk 2 above)
Comment (Edithabich) [reply WindowsForms/MessageBox/Overview.md:3]: resolved

**Pull request closed without merging**